src/visions/relations/relations.py

```diff
@@ -46,9 +46,11 @@ class TypeRelation:
 
     related_type = attr.ib()
     inferential: bool = attr.ib()
-    transformer: Callable[[T, dict], T] = attr.ib(converter=multimethod, repr=func_repr)
+    transformer: Callable[[T, dict], T] = attr.ib(
```

Comment:
Typing was previously working, how come it has to be ignored now?

Reply:
```
src\visions\relations\relations.py:49: error: Unsupported converter, only named functions and types are currently supported
src\visions\relations\relations.py:51: error: Unsupported converter, only named functions and types are currently supported
```
https://github.com/python/mypy/issues/6172

Reply:
that's a bummer, thanks for the link though.